.github/workflows/it-tests.yml

```diff
@@ -18,26 +18,14 @@ jobs:
           credentials_json: ${{ secrets.GCP_CREDENTIALS }}
       - name: cache SBT
         uses: coursier/cache-action@v6
-      - name: Java 11 setup
+      - name: Java 17 setup
         uses: actions/setup-java@v3
         with:
           distribution: temurin
-          java-version: 11
```

Comment:
Should have been
```yaml
java-version: ${{matrix.java}}
```

Reply:
@RustedBones why didn't you apply this change, but set only Java 17?

Reply:
There are come concurrency issue on the BigTable integration test. And IMHO we should be fine by cross testing on unit test.

Reply:
it appears we never had cross-java testing. Maybe cross-scala in unit test, cross-java in IT?

Reply:
We do have cross-java in unit-testing
![image](https://user-images.githubusercontent.com/2845540/224648896-f6980021-924a-4e90-8e80-da4b1138ddfc.png)